scripts/release.bash

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+set -eEuo pipefail
+
+# TODO; more available arguments for this script
+# instead of hardcoded values
+
+function show_help {
+    cat - <<HEREDOC
+Usage:
+
+    ${0##*/} -v VERSION [-n] [-d]
```

Comment:
-v isn't documented below; RPM_LIFECYCLE and DEB_COMPONENT don't seem to be implemented

Reply:
RPM/DEB aren't needed here. Release is a simple helper to put the tag consistently and push, that's it.

I'll document the -v though